### ISSUE TITLE
Turtorial tensorflow addons version mismatch

### ISSUE THREAD

**Issue by fsx950223** [tensorflow/addons#1945]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):colab
- TensorFlow version and how it was installed (source or binary):binary
- TensorFlow-Addons version and how it was installed (source or binary):binary
- Python version:3.7
- Is GPU used? (yes/no):yes

**Describe the bug**
Default tensorflow-addons in turtorial is 0.8.3 with tensorflow==2.2.0
The tensorflow-addons version should >= 0.9.1 according to https://github.com/tensorflow/addons#c-custom-op-compatibility
**Code to reproduce the issue**
https://www.tensorflow.org/addons/tutorials/tqdm_progress_bar
Provide a reproducible test case that is the bare minimum necessary to generate the problem.

**Other info / logs**

Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by seanpmorgan** [comment]
Waiting on: https://github.com/googlecolab/colabtools/issues/1243

**Comment by seanpmorgan** [comment]
Closing since new issue is filed to fix this:
https://github.com/tensorflow/addons/issues/2124
